adders/drizzle/config/tests.js

```diff
@@ -0,0 +1,100 @@
+import { defineAdderTests } from "@svelte-add/core";
+import { options } from "./options";
+
+const defaultOptionValues = {
+    sqlite: options.sqlite.default,
+    mysql: options.mysql.default,
+    postgresql: options.postgresql.default,
+    docker: options.docker.default,
+};
+
+export const tests = defineAdderTests({
+    options,
+    optionValues: [
+        { ...defaultOptionValues, database: "sqlite", sqlite: "better-sqlite3" },
+        { ...defaultOptionValues, database: "sqlite", sqlite: "libsql" },
+        { ...defaultOptionValues, database: "mysql", mysql: "mysql2", docker: true },
+        { ...defaultOptionValues, database: "postgresql", postgresql: "postgres.js", docker: true },
+    ],
+    runSynchronously: true,
```

Comment:
Is there a reason for executing them synchronously? If yes, could you please add a comment indicating why this is the case similar to `storybook`?